app/views/govuk_publishing_components/components/_chat_entry.html.erb

```diff
@@ -0,0 +1,47 @@
+<%
+  add_gem_component_stylesheet("chat-entry")
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
+  href ||= nil
+  heading_text ||= nil
+  description_text ||= nil
+  border_top ||= false
+  border_bottom ||= false
+
+  classes = %w[gem-c-chat-entry]
+  classes << "gem-c-chat-entry--border-top" if border_top
+  classes << "gem-c-chat-entry--border-bottom" if border_bottom
+%>
+
+<%= content_tag("div", class: classes ) do %>
```

Comment:
Non-blocking: have you considered using the [component wrapper helper](https://github.com/alphagov/govuk_publishing_components/blob/main/docs/component-wrapper-helper.md) for this component?

Reply:
I did consider it, but initially thought I might not need it since I was only adding class names. However, I’ve changed my mind and decided to use it after all, as it will be useful if I need to add other attributes, which is quite likely